ResourceManager/GitExtensionsControlInitialiser.cs

```diff
@@ -15,7 +15,11 @@ internal sealed class GitExtensionsControlInitialiser
 
         public GitExtensionsControlInitialiser(GitExtensionsFormBase form)
         {
-            ThreadHelper.ThrowIfNotOnUIThread();
+            if (form.GetType().Name != "BugReportForm")
+            {
+                ThreadHelper.ThrowIfNotOnUIThread();
+            }
```

Comment:
As a quick fix - may be wrap it like this? 
```suggestion
#if DEBUG
            if (form.GetType().Name != "BugReportForm")
            {
                ThreadHelper.ThrowIfNotOnUIThread();
            }
#endif
```

Reply:
> As a quick fix - may be wrap it like this?
> `#if DEBUG`

Why should we omit this in the Release build? What would it quick-fix? Did you mean it without the `if`?
This would replace one code smell with another (slightly less bad).

Reply:
Yes, you're right. I am not sure what I was thinking...